easybuild/easyconfigs/r/RAVEL/RAVEL-1.3.5-foss-2019b-R-3.6.2.eb

```diff
@@ -0,0 +1,91 @@
+easyblock = 'Bundle'
+
+name = 'RAVEL'
+# Listed as 1.3.5 in DESCRIPTION
+version = '1.3.5'
+versionsuffix = '-R-%(rver)s'
+local_git_commit = '7f07ea2'
+
+homepage = 'https://github.com/Jfortin1/RAVEL'
+description = """RAVEL is an R package that combines the preprocessing and statistical analysis of
+ magnetic resonance imaging (MRI) datasets within one framework. Users can start with raw images in
+ the NIfTI format, and end up with a variety of statistical results associated with voxels and
+ regions of interest (ROI) in the brain. RAVEL stands for Removal of Artificial Voxel Effect by
+ Linear regression, the main preprocessing function of the package that allows an effective
+ removal of between-scan unwanted variation."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('R', '3.6.2'),
+    ('R-bundle-Bioconductor', '3.10'),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_list = [
+    ('RNifti', '1.3.0', {
+        'checksums': ['de1629e1672d94ef74404981728d4573abba20d1ef303058d54c9816d3c120ff'],
+    }),
+    ('oro.nifti', '0.11.0', {
+        'checksums': ['5193465f6b8a5087d4bc0d6478ed377afd984bd9bbab29cc255b80fb0462b4ce'],
+    }),
+    ('neurobase', '1.29.0', {
+        'checksums': ['0aa6aa2f3993f20f76331cfcb42d1686402b8bbd74030406a1a734f23069c19b'],
+    }),
+    ('WhiteStripe', '2.3.2', {
+        'checksums': ['cf96a19fa6cec6516b5948dba73a06d866c1a376e189d6bf0302d9af147dfc4f'],
+    }),
+    ('fslr', '2.24.1', {
+        'checksums': ['282e0579060117d82118e307018fbd8ca861ca7f9e3bd68e9c344fb4e5c652df'],
+    }),
+    ('ITKR', '0.5.3.3.0', {
```

Comment:
This fails for me due to a too old system CMake. Investigating this builds ITK - https://github.com/stnava/ITKR/blob/master/configure

Ideally, if it is possible, we'd use an EB ITK instead at this point. Or, this EC will require a bunch of extra deps / build deps to support building ITK.

I've not checked, but I'd suspect something similar for ANTs as well.

Reply:
It's unlikely I'll be able to clean this up in the short-term, so closing.